hs_core/templates/search/indexes/hs_core/baseresource_text.txt

```diff
@@ -25,9 +25,15 @@
 {% for res in object.metadata.time_series_results %}{{ res.sample_medium|discoverable }} {{ res.units_name }} {{ res.units_type }}  {% endfor %}
 {% for f in object.logical_files %} 
 {{ f.get_discovery_content_type }} {{ f.metadata.cellinformation.name|discoverable }}  
-{% for v in f.metadata.variables.all %}{{ v.name|discoverable }} {% endfor %} 
+{% for v in f.metadata.variables.all %}{{ v.name|discoverable }} {{ v.code|discoverable }} {% endfor %} 
 {% for b in f.metadata.bandInformations %}{{ b.variableName|discoverable }} {% endfor %}
+{% for m in f.metadata.methods %} {{ m.description|discoverable }} {% endfor %}
+{% for m in f.metadata.sample_mediums %} {{ m|discoverable }} {% endfor %}
+{% for s in f.metadata.sites %} {{ s.name|discoverable }} {% endfor %}
 {% if f.get_discovery_content_type == "Time Series" %}
-{% for v in f.metadata.variables %}{{ v.variable_name|discoverable }} {% endfor %} 
+{% for s in f.metadata.methods %}{{ s.method_description|discoverable }} {% endfor %}
+{% for s in f.metadata.sites %}{{ s.site_name|discoverable }} {% endfor %}
+{% for v in f.metadata.variables %}{{ v.variable_name|discoverable }} {{ v.variable_code|discoverable }} {% endfor %} 
+{% for r in f.metadata.time_series_results %} {{ r.sample_medium }} {{ r.method }} {{ r.units_name }} {{ r.units_type }} {% endfor %}
```

Comment:
Maintaining this file when metadata fields are created/deleted/updated seems tedious.  It seems tedious even trying to get each metadata value into this document initially.  Are there plans to put all metadata values into solr?

Reply:
This particular file consists only of the metadata values of type text. It's useless to put geospatial values in text search. That's why this template is selective.